src/getCommitMeaning.test.ts

```diff
@@ -40,6 +40,17 @@ describe("getCommitMeaning", () => {
 		["chore(deps): release", "release"],
 		["chore(deps): release 1.2.3", "release"],
 		["chore(deps): release v1.2.3", "release"],
+		// Breaking changes are meaningful no matter the type of the commit.
+		["chore!: message", "meaningful"],
+		["docs!: message", "meaningful"],
+		["style!: message", "meaningful"],
+		["chore!: release", "meaningful"],
+		["chore: bump\n\nBREAKING CHANGE: breaks things", "meaningful"],
+		["chore: bump\n\nBREAKING-CHANGE: breaks things", "meaningful"],
+		["docs: bump\n\nBREAKING CHANGE: breaks things", "meaningful"],
+		["docs: bump\n\nBREAKING-CHANGE: breaks things", "meaningful"],
+		["style: bump\n\nBREAKING CHANGE: breaks things", "meaningful"],
+		["style: bump\n\nBREAKING-CHANGE: breaks things", "meaningful"],
```

Comment:
[Testing] Thinking about test edge cases...

* `!` in the commit body (so not a breaking/meaningful change)
* Footer note other than `BREAKING CHANGE`
* Multiple footer notes